packages/components/src/controllers/useSizeOverride.ts

```diff
@@ -0,0 +1,94 @@
+import { makeController } from "@arcgis/lumina/controllers";
+
+/**
+ * Axis used when applying a size override.
+ * "inline" maps to width, "block" maps to height.
+ */
+export type SizeAxis = "inline" | "block";
+
+/**
```

Comment:
I think we can omit this comment since it's not adding much. Applies to `UseSizeOverride`.